server/etcdserver/api/etcdhttp/health.go

```diff
@@ -240,8 +281,36 @@ func (reg *CheckRegistry) InstallHttpEndpoints(lg *zap.Logger, mux *http.ServeMu
 	}
 }
 
+func (reg *CheckRegistry) recordMetrics(name string, status string) {
+	val := 0
+	if status == HealthStatusSuccess {
+		val = 1
+	}
+	// record the metrics for the root path.
+	if len(name) == 0 {
+		rootHealthCheckGauge.With(prometheus.Labels{
+			"type": reg.path,
```

Comment:
How about stripping the leading forward slash for `type`? `livez` instead of `/livez`?

Reply:
done